typings/index.d.ts

```diff
@@ -0,0 +1,232 @@
+import * as React from 'react';
+
+interface AnyObject {
+  [key: string]: any;
+}
+
+interface CommonProps<T = any> {
+  instanceRef?: (ref: React.Ref<T>) => void;
+  onLoad?: (ymaps: YMapsApi) => void;
+  onError?: (error: Error) => void;
+  modules?: string[];
+}
+
+export interface YMapsApi extends AnyObject {}
+
+export interface YMapsProps extends AnyObject {
+  version?: string;
+  enterprise?: boolean;
+  query?: {
+    lang?: 'tr_TR' | 'en_US' | 'en_RU' | 'ru_RU' | 'ru_UA' | 'uk_UA';
+    apikey?: string;
+    coordorder?: 'latlong' | 'longlat';
+    load?: string;
+    mode?: 'release' | 'debug';
+    csp?: boolean;
+    ns?: string;
+  };
+  children?: React.ReactNode;
+  preload?: boolean;
+}
+
+export interface MapState extends AnyObject {
+  bounds?: number[][];
+  center?: number[];
```

Comment:
I suppose we should make **center** and **zoom** required params, cause without them we can't get map

Reply:
It's either bounds or center/zoom combo. Is it possible to express someting like this in ts?

Reply:
I do not know for sure if it's possible, but I hope it is. I'll look at solutions on this case.

Reply:
So, @gribnoysup, @mmarkelov, what do you think, do we need the detailed definitions of `ymaps` property or not?

Reply:
So, I added a test version of required `bounds` or `center`/`zoom` map state properties in 1a7074b, edits are welcome.

By the way, I still insist on not making everything declared, because [@types/yandex-maps](https://www.npmjs.com/package/@types/yandex-maps) is not as stable as it needs to be and outdated in some parts.

Reply:
@zalishchuk I'm not a big typescript user so I'll trust you on that 👍

This PR is already pretty awesome even without ymaps types 😁

Reply:
@zalishchuk youp! I also don't have so much experience with TS,  and I also think that making everything declared is overhead and unnecessary, we can make changes if we need them.